python/perspective/perspective/tests/handlers/test_starlette_handler.py

```diff
@@ -64,6 +65,14 @@ async def websocket_client(self):
         """
         return await websocket(CLIENT, "/websocket")
 
+    @asynccontextmanager
+    async def managed_websocket_client(self):
```

Comment:
I wrote my test first in the style of the others. But when I undid the fix to confirm it would make my test red, it also made the test hang: when the test is red, `await get_hosted_table_names()` throws, which exits the test before the `client.terminate()` call.

The other tests in this file would also hang if they broke. I can rewrite the other tests in this style if you want

Reply:
It was so easy I went ahead and made the change and split it out into a separate commit. Let me know if you'd like any changes